src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CreateCaseService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.config.CaseFlagsConfiguration;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.singletonMap;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.YES_VALUE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APPLICANT_REPRESENTED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.MINI_FORM_A;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CreateCaseService {
+
+    private final AuthTokenGenerator authTokenGenerator;
+    private final OnlineFormDocumentService onlineFormDocumentService;
+    private final IdamService idamService;
+    private final CoreCaseDataApi coreCaseDataApi;
+
+    private final CaseFlagsConfiguration caseFlagsConfiguration;
+
+    public void draftContestedMiniFormA(CallbackRequest callbackRequest, String userAuthorisation) {
+        Map<String, Object> caseData = callbackRequest.getCaseDetails().getData();
+        CaseDocument document = onlineFormDocumentService.generateDraftContestedMiniFormA(userAuthorisation, callbackRequest.getCaseDetails());
+        caseData.put(MINI_FORM_A, document);
+        if (!idamService.isUserRoleAdmin(userAuthorisation)) {
+            log.info("other users.");
+            caseData.put(APPLICANT_REPRESENTED, YES_VALUE);
```

Comment:
what logic determines if Applicant represented?

Reply:
that method was left behind by mistake as I initially was going to refactor the aboutToSubmit as well but decided not to. removing it now thanks